packages/yarn-plugin-ado-auth/package.json

```
@@ -0,0 +1,41 @@
+{
+  "name": "yarn-plugin-ado-auth",
```

We should not be putting any pacakges in global scope anymore, but in the @microsoft scope.
```suggestion
  "name": "@microsoft/yarn-plugin-ado-auth",
```